sdk/nodejs/runtime/resource.ts

```diff
@@ -92,7 +91,7 @@ interface ResourceResolverOperation {
 /**
  * Get an existing resource's state from the engine.
  */
-export function getResource(res: Resource, props: Inputs, custom: boolean, urn: string): void {
+export function getResource(res: Resource, parent: Resource | undefined, props: Inputs, custom: boolean, urn: string): void {
```

Comment:
We believe nothing outside the code already updated in this PR would call this func? So we don't care if this is breaking?

Reply:
Yeh we don't expect things outside the SDK to call into runtime.